pyomo/contrib/iis/tests/trivial_mis.py

```diff
@@ -0,0 +1,15 @@
+import pyomo.environ as pyo
+
+m = pyo.ConcreteModel("Trivial Quad")
+m.x = pyo.Var([1, 2], bounds=(0, 1))
+m.y = pyo.Var(bounds=(0, 1))
+m.c = pyo.Constraint(expr=m.x[1] * m.x[2] == -1)
+m.d = pyo.Constraint(expr=m.x[1] + m.y >= 1)
+
+from pyomo.contrib.iis.mis import compute_infeasibility_explanation
+
+# if IDAES is installed, compute_infeasibility_explanation doesn't need to be passed a solver
```

Comment:
Because we want you to remove the dependency on IDAES, please also adjust this message.